css/index.css

```diff
@@ -3,83 +3,132 @@
   width: 1000px;
 }
 body {
-  background-color: rgba(255, 228, 196, 0.281);
+   background-color: #efe5dcd8; 
   font-family: "Poppins", sans-serif;
+  padding: 25px;
+  /* background-image: url("../img/beige.jpeg") ; */
 }
 h1 {
-  color: chocolate;
+  color: #463f3a;
   font-size: 50px;
   font-weight: 60;
   text-align: center;
   font-family: cursive;
+  padding: 25px 0;
+  margin-bottom: 25px;
 }
 h2 {
-  color: chocolate;
+  color: #463f3a;
   font-size: 35px;
   font-weight: 200;
   font-family: fantasy;
+  padding: 25;
 }
 p {
-  color: rgba(160, 81, 45, 0.925);
-
+  color: #8a817c;
   font-size: 20px;
   font-family: monospace;
+  padding: 25px;
 }
 a {
-  color: rgba(160, 81, 45, 0.925);
+  color: #8a817c;
 }
 
 section {
   text-align: center;
-  margin: 10px 20px 30px 40px;
+  margin: 25px;
 }
 h1,
 h2,
 p {
-  background-color: blanchedalmond;
+  background-color: #efe5dc;
+  
+
 }
 ul {
   list-style-type: none;
-  margin: 0;
+  margin: 10px 20px 30px 40px;
   padding: 0;
+  font-family: monospace;
+  font-size: 17px;
 }
 li {
   display: inline;
-  color: rgba(160, 81, 45, 0.925);
+  color: #8a817c;
 }
 a {
   display: block;
   padding: 8px;
   background-color: whitesmoke;
   text-decoration: none;
 }
+
 img {
+  float: right;
   border-radius: 8%;
   width: 25%;
   height: auto;
-  box-shadow: 10px 5px 10px grey;
+  box-shadow: 10px 5px 10px #8a817c;
 }
-.aboutMe {
-  padding: 1em;
+.about-me {
+  padding: 25px;
+}
+nav > ul{
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
 }
 #navbar a {
   display: inline;
   padding: 8px;
-  background-color: whitesmoke;
+  background-color: #efe5dc;
   text-decoration: none;
+  text-align: center;
 }
 ul.a {
   list-style-type: circle;
 }
 .connect > a:link {
-  color: darkgoldenrod;
+  color: #463f3a;
 }
 a:visited {
-  color: darkgoldenrod;
+  color: #463f3a;
 }
 a:hover {
-  color: darkgoldenrod;
+  color: #463f3a;
 }
 a:active {
-  color: darkgoldenrod;
+  color: #463f3a;
+}
+
+.flex-container {
```

Comment:
replace the class `.flex-container` with the code below

.flex-container {
  display: grid;
  grid-template-rows: auto;
  grid-gap: .5rem;
  margin: .5rem 0;
}